homeassistant/components/backblaze/__init__.py

```diff
@@ -63,7 +68,22 @@ def _authorize_and_get_bucket_sync() -> Bucket:
             entry.data[CONF_KEY_ID],
             err,
         )
-        raise ConfigEntryError(
+        # Create repair issue for user to address
+        ir.async_create_issue(
+            hass,
+            DOMAIN,
+            f"bucket_access_restricted_{entry.entry_id}",
+            is_fixable=True,
+            issue_domain=DOMAIN,
+            severity=ir.IssueSeverity.ERROR,
+            translation_key="bucket_access_restricted",
+            translation_placeholders={
+                "title": entry.title,
+                "bucket_name": err.bucket_name,
+                "entry_id": entry.entry_id,
+            },
+        )
```

Comment:
Maybe add helpers in `repairs.py` for creating the repairs to avoid duplicating it?